group-generators/helpers/data-providers/poap/factory-schema.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "POAP",
+  "iconUrl": "",
+  "providerClassName": "POAPProvider",
+  "functions": [
+    {
+      "name": "Attenders of specific events",
+      "functionName": "queryEventsTokenOwners",
+      "description": "Return the list of Ethereum accounts that attended a list of POAP events",
+      "args": [
+        {
+          "name": "event Ids",
+          "type": "(number | string)[]",
```

Comment:
"(number | string)[]" does not exists in the design system, let use string[]

Reply:
done